RELEASE/scripts/autoscend/paths/auto_path_util.ash

```diff
@@ -185,6 +185,11 @@ boolean auto_buySkills()  // This handles skill acquisition for general paths
 		{
 			visit_url("guild.php?action=buyskill&skillid=6", true);
 		}
+		if((my_level() >= 12) && (my_meat() >= 500) && !have_skill($skill[Deft Hands]) && 
+		(!have_skill($skill[Ambidextrous Funkslinging]) || (item_amount($item[beehive]) == 0 && item_amount($item[Time-Spinner]) == 0)))	//safe flyering
+		{
+			visit_url("guild.php?action=buyskill&skillid=6", true);
```

Comment:
1. isn't skill ID 6 [Mad Looting Skillz] as per line 186
2. needs a comment saying which skill you are getting here. since you are checking for have_skill on two different skills so it is unclear which of them you are meaning to buy here